### PR TITLE
Add Anchor supporting and cache mechanism section (#2334)

### LINKED ISSUE #2334: Supporting of anchor in the Getting-started page sections of BitPlatform website and also needs cache mechanism section.
The getting-Started page of the Bitplatform website currently does not support the Anchor also this document doesn't have any section about cache mechanism.

### PR DESCRIPTION
**Pull request by MeysamBahadori**
This closes #2334

### CODE REVIEW

File: src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor [hunk 1]
```diff
@@ -49,7 +49,7 @@
                 </li>
                 <li>
                     Create a new project using the Template installed
-                    <pre class="code"><code>dotnet new bit-todo</code></pre>
+                    <pre class="code"><code>dotnet new Bit.TodoTemplate</code></pre>
```
Comment (ysmoradi) [inline src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor]: ?

File: src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ﻿
 <section class="page-section getting-started-step-section">
 
-    <div class="step-body-card">
+    <div id="getting-started" class="step-body-card">
```
Comment (ysmoradi) [inline src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor:4]: assigning `id` to the `div` will result into the `anchor`?

**Review by ysmoradi — CHANGES_REQUESTED** [review]

File: src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor (hunk 1 above)
Comment (msynk) [reply src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor]: this change should be rolled back

File: src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor (hunk 2 above)
Comment (msynk) [reply src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor:4]: yes, the spec:
https://www.w3.org/TR/html4/struct/links.html#h-12.2.3

**Review by ysmoradi — APPROVED** [review]

File: src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor (hunk 2 above)
Comment (MeysamBahadori) [reply src/WebSite/Bit.Platform.WebSite/Web/Components/GettingStarted.razor:4]: Yes, do you think any change need?

**Review by msynk — APPROVED** [review]

**Pull request merged**